docs/install/mingw.mdx

```diff
@@ -8,115 +8,47 @@ title: Build on Windows (MinGW-w64)
-7. [WinPcap developer's pack](https://www.winpcap.org/devel.htm) **OR** [Npcap SDK](https://nmap.org/npcap/guide/npcap-devguide.html) - containing the `wpcap` library PcapPlusPlus is linking with plus relevant `h` files.
+1. `CMake` which can be downloaded from: <https://cmake.org/download/>
+2. Download and run MinGW-w64 installer from here: <https://sourceforge.net/projects/mingw-w64/>
+3. Make sure that MinGW-w64 `bin` directory is in your `PATH` environment variable, for example:
+   ```console
+   PATH=%PATH%;C:\mingw-w64\i686-8.1.0-posix-dwarf-rt_v6-rev0\mingw32\bin
+   ```
+4. [WinPcap developer's pack](https://www.winpcap.org/devel.htm) **OR** [Npcap SDK](https://nmap.org/npcap/guide/npcap-devguide.html) - containing the `wpcap` library PcapPlusPlus is linking with plus relevant `.h` files.
    1. WinPcap developer's pack can be downloaded from here: <https://www.winpcap.org/devel.htm>
    2. Npcap SDK can be downloaded from here: <https://nmap.org/npcap/#download>
 
-:::danger x64 is not supported
-Please notice that **x64** compilation is not supported (and will not work) on MinGW-w64!
-:::
+## Build
 
-## Configuration
-
-Run the `configure-windows-mingw.bat` batch file from PcapPlusPlus main directory. The script creates a makefile `mk\PcapPlusPlus.mk` that contains paths for 3rd-party libraries being used to build PcapPlusPlus.
-
-Ths script has two modes of operation:
-
-- Wizard mode - the script will walk you through the paths and parameters you need to provide
-- Params mode - all parameters are provided in the script command line
-
-In wizard mode the script will ask you for MinGW-w64 location (for example: `C:\i686-8.1.0-posix-dwarf-rt_v6-rev0`) and also for WinPcap developer's pack / Npcap SDK location (for example: `C:\WpdPack` or `C:\Npcap-SDK`).
-
-Here is an example of running the script:
-
-```text
-******************************************
-PcapPlusPlus Windows configuration script
-******************************************
-
-If mingw-w64 is not installed, please download and install it from here:
-https://www.mingw-w64.org/downloads/#mingw-builds
-
-Please specify mingw-w64 installed path (for example: C:\i686-8.1.0-posix-dwarf-rt_v6-rev0): C:\i686-8.1.0-posix-dwarf-rt_v6-rev0
-
-
-MSYS2 is required for compiling PcapPlusPlus.
-If MSYS2 are not installed, please download and install it from: https://www.msys2.org/
-
-Please specify MSYS2 installed path: C:\msys64
-
-
-WinPcap or Npcap SDK is required for compiling PcapPlusPlus.
-For downloading WinPcap SDK (developer's pack) please go to https://www.winpcap.org/devel.htm
-For downloading Npcap SDK please go to https://nmap.org/npcap/#download
-
-Please specify WinPcap/Npcap SDK path: C:\Npcap-SDK
-
-PcapPlusPlus configuration is complete. Files created (or modified): mk\platform.mk, mk\PcapPlusPlus.mk
+```console
+mkdir build
+cd build
+cmake .. -G "MinGW Makefiles" -DPCAP_ROOT=<NpcapSDK_or_WinPcapDevPack_Directory>
```

Comment:
can be done in one line 
```
cmake -G "MinGW Makefiles" -DPCAP_ROOT=<NpcapSDK_or_WinPcapDevPack_Directory> -S . -B build 
```

Reply:
Also this only configure I think it's missing the 

```
cmake --build build -j
```

Reply:
> Also this only configure I think it's missing the
> 
> ```
> cmake --build build -j
> ```

I'm not sure I understand this comment... 🤔

Reply:
In your first version it was missing the 
```
cd build
make
```

I think instead of calling make directly, it's better to do 
```
cmake --build build -j
```
To tell CMake to run the build of the build folder. This will make the documentation almost identifical if you use Make, Ninja, Visual Studio or another generator

Reply:
Sure, fixed here: https://github.com/PcapPlusPlus/pcapplusplus.github.io/pull/48/commits/8f44015495d0448ea0eba48c620a4ad605277800

I added the 2 options so users can choose what they prefer